### DISCUSSION TITLE
Better release notes

### DISCUSSION THREAD

**Discussion by erbridge** [vercel/next.js#11991] [Help]
It's currently really difficult to determine what's changed between versions. It looks like releases are created based on titles of pull requests, which are often lacking in detail, meaning a review of what's changed, and therefore might imply changes in the consumer, requires a review of every PR that went in to a release.

It would be really helpful if those changes were, at a minimum, categorized to make them more scannable, e.g. using [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

**Comment by rafaelalmeidatk** [comment]
I think the release notes text is built with a bot, but it would be awesome if we could at least remove the PRs to the examples since they are not relevant. Perhaps the bot could ignore the PRs with the Example label

**Reply by timneutkens** [reply]
We have some ideas to automate this. Just haven't gotten around to implementing this as the backlog of features/issues on the roadmap is quite long currently. If anyone wants to work on it feel free to reach out.

**Reply by rafaelalmeidatk** [reply]
@timneutkens I would like to help with this! Where do we discuss it?

**Reply by Janpot** [reply]
Usually, I find projects that use a `CHANGELOG.md` more ergonomic to upgrade than ones that use release notes. It's just easier to scan a single file than to have to navigate between pages of Github release notes. Just copying the github release notes in a `CHANGELOG.md` would already be quite helpful.

**Reply by timneutkens** [reply]
> @timneutkens I would like to help with this! Where do we discuss it?

Feel free to reach out on https://twitter.com/timneutkens
